msgraph-sdk-raptor-compiler-lib/identifiers.json

```
@@ -535,5 +535,14 @@
   },
   "serviceHealth": {
     "_value": "serviceHealth"
+  },
+  "externalConnectors.externalConnection": {
+    "_value": "externalConnectors.externalConnection",
+    "externalConnectors.connectionOperation": {
+      "_value": "externalConnectors.connectionOperation"

```

naming pattern for placeholders:
```suggestion
      "_value": "externalConnectors.externalConnection_externalConnectors.connectionOperation"
```